models/balancer/balancer_schema.yml

```diff
@@ -151,3 +151,30 @@ models:
       - name: recipient
       - name: contract_address
 
+#  - name: balancer_bpt_prices
+#    meta:
+#      blockchain: ethereum, arbitrum, optimism, polygon, gnosis
+#      sector: dex
+#      contributors: thetroyharris, victorstefenon
+#    config:
+#      tags: ['balancer', 'amm', 'dex', 'bpt', 'prices']
+#    description: >
+#      Balancer Pool Token (BPT) hourly median price by pool on Balancer, an automated portfolio manager and trading platform.
+#    tests:
+#      - dbt_utils.unique_combination_of_columns:
+#          combination_of_columns:
+#            - blockchain
+#            - hour
+#            - contract_address
+#    columns:
+#      - *blockchain
+#      - &hour
+#        name: hour
+#       description: 'UTC event block time truncated to the hour mark'
+#        tests:
+#          - not_null
+#      - *contract_address
+#     - &median_price
+#        name: median_price
+#        description: 'BPT median price'
```

Comment:
Hi @Hosuke, I commented out the schema for that spell because that spell, which was getting the balancer_bpt_prices from every blockchain and putting it into 1 table, was returning an "exceeds cluster capacity" error and is not as important as updating the "balancer_bpt_prices" for each blockchain individually.

However, I would love to get that spell working eventually so if you have any thoughts / suggestions they would be very much appreciated!

Reply:
Thank you for the feedback.

Reply:
cc. @augustog for the feedback.